packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -537,4 +533,20 @@ export class Cluster extends ClusterBase {
       target: this,
     });
   }
+
+  private validateNodeCount(clusterType: ClusterType, numberOfNodes?: number): number | undefined {
```

Comment:
rename to `parseNodeCode` because it returns a value. Usually "validateXxxx" methods return `void`.

Reply:
`validateXxxx` methods usually return void, and `parseXxxx` methods usually parse something. Neither really fits here IMO. I personally prefer "validate" here as describing more of the "main" intent rather than "parse", which doesn't describe the method at all.

That being said, I see only a few "validate" methods with return types (one of which was also my fault), and lots of "parse" methods that don't actually parse anything, so I guess that's the defined convention at this point. Would something like `validateAndReturnXxx` or `validateOrDefault` or something like that be any better?